src/hooks/useCrossHair.tsx

```diff
@@ -9,10 +9,27 @@ import { ControllerHookOptions, DualAxisOptions } from './types';
 export interface UseCrossHairOptions
   extends ControllerHookOptions,
     DualAxisOptions {
+  /**
+   * Line color
+   * @default "black"
+   *
+   * */
   color?: CSSProperties['stroke'];
+  /**
+   * Line style
```

Comment:
Try to be a little bit more detailed than what can already be understood from the prop's name.

```suggestion
   * Additional style to apply to the crosshair's lines
```